system-test/compute.js

```diff
@@ -693,70 +694,44 @@ describe('Compute', () => {
 
   describe('rules', () => {
     const RULE_NAME = generateName('rule');
-    const rule = compute.rule(RULE_NAME);
-    const service = compute.service(generateName('service'));
-    const INSTANCE_GROUP_NAME = generateName('instance-group');
-    const HEALTH_CHECK_NAME = generateName('health-check');
-
-    // To create a rule, we need to also create a TargetHttpProxy and UrlMap.
-    // Until they are officially supported by google-cloud-node, we make manual
-    // requests to create and delete them.
-    const TARGET_PROXY_NAME = generateName('target-proxy');
-    const URL_MAP_NAME = generateName('url-map');
+    const NETWORK_NAME = generateName('network-name');
+    const SUBNETWORK_NAME = generateName('subnetwork-name');
+    const BACKEND_SERVICE_NAME = generateName('backend-service-name');
 
     before(async () => {
-      await createService(service.name, INSTANCE_GROUP_NAME, HEALTH_CHECK_NAME);
-      await createUrlMap({
-        name: URL_MAP_NAME,
-        defaultService: 'global/backendServices/' + service.name,
-      });
-      await createTargetProxy({
-        name: TARGET_PROXY_NAME,
-        urlMap: 'global/urlMaps/' + URL_MAP_NAME,
-      });
-      await create(rule, {
-        target: 'global/targetHttpProxies/' + TARGET_PROXY_NAME,
-        portRange: '8080',
-        IPProtocol: 'TCP',
-      });
+      execSync(
+        `gcloud compute networks create ${NETWORK_NAME} --subnet-mode=custom`
+      );
```

Comment:
```suggestion
      const [network, operation] = await compute.createNetwork(NETWORK_NAME, {autoCreateSubnetworks: false});
      await operation.promise();
```

^ I'm not sure that's equivalent. Just a guess from the [docs](https://cloud.google.com/compute/docs/reference/rest/v1/networks#resource): 'When set to true, the VPC network is created in auto mode. When set to false, the VPC network is created in custom mode.'